src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Deposit.tsx

```diff
@@ -172,8 +174,17 @@ export const Deposit: React.FC<DepositProps> = ({
   } = useGetThorchainSaversDepositQuoteQuery({
     asset,
     amountCryptoBaseUnit: toBaseUnit(inputValues?.cryptoAmount, asset.precision),
+    enabled: !isRunePool,
   })
 
+  const memo = useMemo(() => {
+    if (isRunePool) return RUNEPOOL_DEPOSIT_MEMO
```

Comment:
Same comment re: adding me to the `Action` Union

Reply:
Did in [19db6af](https://github.com/shapeshift/web/pull/7440/commits/19db6afed2d77fec1f3cb46386d3487d42dbb41e)